### PR TITLE
CSP

### PR DESCRIPTION
**Pull request by kof**

### CODE REVIEW

**Comment by Graham42** [comment]
looks like IE doesn't support `remove` so the cleanup of the nonce test fails. https://stackoverflow.com/a/37477096

File: tests/functional/sheet.js [hunk 1]
```diff
@@ -84,18 +84,25 @@ describe('Functional: sheet', () => {
...
+      document.head.appendChild(nonceMetaTag)
+
       sheet = jss.createStyleSheet().attach()
       style = getStyle()
     })
 
     afterEach(() => {
       sheet.detach()
+
+      nonceMetaTag.remove()
```
Comment (kof) [inline tests/functional/sheet.js]: yeah, please use parentNode.removeChild

Comment (kof) [reply tests/functional/sheet.js]: fixed it

**Comment by Graham42** [comment]
Green tests! Sweet!

**Comment by kof** [comment]
merged!!!!

**Pull request merged**